nomad/jitsi_packs/packs/jitsi_oscar/templates/_cloudprober_config.tpl

```diff
@@ -92,32 +95,101 @@ probe {
   interval_msec: 10000
   timeout_msec: 2000
 }
+
 [[ end -]]
 [[ if var "enable_shard" . -]]
-# probes health of all shards in all datacenters using public IP
+# probes health of all shards via their signal-sidecars, in all datacenters, using public IP
 probe {
   name: "shard"
   type: HTTP
   targets {
-    {{ $shard_count := 0 }}
-    {{ range $dc := datacenters }}{{ $dc_shards := print "signal-sidecar@" $dc }}{{ range $shard := service $dc_shards -}}
-      {{ $shard_count = add $shard_count 1 }}
+    {{ $shard_count := 0 -}}
+    {{ range $dc := datacenters -}}{{ $dc_shards := print "signal-sidecar@" $dc -}}{{ range $shard := service $dc_shards -}}
```

Comment:
I'm curious why we don't use the "signal" service here, which should also catch shards that aren't directly in nomad

Reply:
because i want to query the service port of signal-sidecar

Reply:
that's an interesting question for nomad vs non-nomad shards, tho

Reply:
signal_sidecar_http_port is available in the signal service meta, for shards in nomad.  For non-nomad we have a fixed port.

Reply:
good suggestion, fixed